art/estimators/pytorch.py

```diff
@@ -114,6 +114,21 @@ def loss(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
         """
         raise NotImplementedError
 
+    def clone_for_refitting(self) -> 'PyTorchEstimator':  # lgtm [py/inheritance/incorrect-overridden-signature]
+        """
+        Create a copy of the estimator that can be refit from scratch.
+
+        :return: new estimator
+        """
+        raise NotImplementedError
+
+    def reset(self) -> None:
+        """
+        Resets the weights of the estimator so that it can be refit from scratch.
+
+        """
+        raise NotImplementedError
+
```

Comment:
I think we should not add these lines as they remove the requirement from classes inheriting from this class of implementing these new methods.